swarm/network/stream/visualized_snapshot_sync_sim_test.go

```diff
@@ -218,11 +240,151 @@ func TestSnapshotSyncWithServer(t *testing.T) {
 		Type:    EventTypeSimTerminated,
 		Control: false,
 	}
-	sim.Net.Events().Send(evt)
+	go sim.Net.Events().Send(evt)
 
 	if result.Error != nil {
 		panic(result.Error)
 	}
-	close(quit)
+	//close(quit)
```

Comment:
?

Reply:
This line should be removed as quit channel is removed.